sros2/sros2/api/__init__.py

```diff
@@ -21,6 +21,10 @@
 import sys
 
 from lxml import etree
```

Comment:
linter error :smile:
The cryptography imports should come before the lxml import.

Reply:
Whoops, sorry-- fixed!